samples/Experimental/VulkanRender/CMakeLists.txt

```diff
@@ -1,4 +1,3 @@
-include(CopyBuildProducts)
```

Comment:
Is this no longer required?

Reply:
Not here, it was a dependency for all samples so I moved it into the higher level Samples CMakeLists for accessibility